src/nodedef.cpp

```diff
@@ -1609,6 +1610,9 @@ void ContentFeatures::serializeOld(std::ostream &os, u16 protocol_version) const
 			compatible_param_type_2 = CPT2_FACEDIR;
 		else if (compatible_param_type_2 == CPT2_COLORED_WALLMOUNTED)
 			compatible_param_type_2 = CPT2_WALLMOUNTED;
+
+		if (drawtype == NDT_PLANTLIKE)
```

Comment:
If the protocol version < 28 and param_type_2 = meshoptions then this won't execute.